src/style.css

```diff
@@ -1,3 +1,71 @@
+:root {
+  --main-color:  #00ACDC;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+  background-color: #fff;
+  box-sizing: border-box;
```

Comment:
that's the possible solution

but FYI: it's better to set it in the `*` selector to avoid duplication (that was original comment meaning)
Note that `box-sizing` is the only property allowed to set in `*`

Reply:
@TarasHoliuk  Thank you) But I really don't understand it =((